app/create/index.js

```diff
@@ -1,9 +1,10 @@
 'use strict';
 
 var Angular = require('angular');
-
+var Controller = require('./controller');
```

Comment:
nit: there should be a new line separating third party requires and project requires so add a new line after line 3

Reply:
nit: align equals of both sections independently